com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDTarget.cs

```diff
@@ -846,39 +846,23 @@ static class CorePragmas
         };
 
         // Here are the Pragma Collection we can add on top of the Basic one
-        public static PragmaCollection DotsInstancedInV2Only = new PragmaCollection
+        public static PragmaCollection DotsInstanced = new PragmaCollection
         {
-            { Pragma.InstancingOptions(InstancingOptions.RenderingLayer) },
-            #if ENABLE_HYBRID_RENDERER_V2
+            { Basic },
+            { Pragma.MultiCompileInstancing },
             { Pragma.DOTSInstancing },
+            { Pragma.InstancingOptions(InstancingOptions.RenderingLayer) },
             { Pragma.InstancingOptions(InstancingOptions.NoLodFade) },
```

Comment:
BatchRendererGroup API nowadays supports LOD fade. This line should be removed. Hybrid Renderer doesn't support LOD fade yet, but I would assume that the default value (zero) it feeds to the BRG API is OK. Hybrid Renderer also explicitly doesn't enable the DrawCommand LOD Crossfade bit, meaning that there should be no complications there.